README.md

```diff
@@ -4,11 +4,23 @@ This project provides common APIs for applications that live on vets.gov. This r
 
 ## Developer Setup
 
+### Base Setup
+(TODO: update these instructions as no database currently necessary)
+
 1. Install Ruby 2.3. (It is suggested to use a Ruby version manager such as [rbenv](https://github.com/rbenv/rbenv#installation) and then to [install Ruby 2.3](https://github.com/rbenv/rbenv#installing-ruby-versions)).
 1. Install Bundler to manage dependencies: `gem install bundler`
 1. Setup the database: `bundle exec rake db:migrate`
 1. Start the application: `bundle exec rails s`
 
+### Redis
+
+1. Install Redis (on mac): `brew install redis`
+1. Follow post install instructions a) always have redis running as service, b) manually launch: `redis-server /usr/local/etc/redis.conf`
```

Comment:
let's make sure that this maps to the govcloud supported version specifically
Redis Version 2.8.24
http://docs.aws.amazon.com/AmazonElastiCache/latest/UserGuide/SelectEngine.html